src/Core/NuGet/Core/Package.nuspec

```diff
@@ -11,6 +11,7 @@
     <iconUrl>$image$</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>Application Insights Core API. This package provides core functionality for transmission of all Application Insights Telemetry Types and is a dependent package for all other Application Insights packages. Please install the platform specific package for the best experience.</description>
+    <summary>Application Insights Core API. This package provides core functionality for transmission of all Application Insights Telemetry Types and is a dependent package for all other Application Insights packages. Please install the platform specific package for the best experience. Privacy statement: https://go.microsoft.com/fwlink/?LinkId=512156</summary>
```

Comment:
Description here is too long. I added summary that can be longer than description.

---
In reply to: [76337553](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/297#discussion_r76337553) [](ancestors = 76337553)